Core/Graphics/Source/GraphicsImpl.cpp

```diff
@@ -86,11 +86,14 @@ namespace Babylon
 
         if (!m_state.Bgfx.Initialized)
         {
+            auto& init{m_state.Bgfx.InitState};
+
             // Set the thread affinity (all other rendering operations must happen on this thread).
-            m_renderThreadAffinity = std::this_thread::get_id();
+            // If context is specified, it is checked in an external thread.
+            if (init.platformData.context == nullptr)
+                m_renderThreadAffinity = std::this_thread::get_id();
```

Comment:
Can you clarify what intended usage is being described here?

Reply:
If the device was created externally, I thought that thread affinity should be managed where it was created. Ultimately, it will disappear when multiview is supported...

Reply:
I see. I think that makes sense, though I'm not sure I fully grasp all the implications; and I think it's aligned with the approach we want, though perhaps in a slightly different form. The changes coming in #591 will, I think, likely have significant implications for this because we're pretty dramatically refining the threading model. Under the new paradigm, the "render thread" *is* something that the Graphics object needs to know about in order to enforce that certain calls are executed correctly, but it should be possible for the "render thread" to change identities by calling `DisableRendering` on the old "render thread" and `EnableRendering` on the new one. We're hoping #591 will be able to go in quite soon, likely within a week. Since your change is also concerned with threading, at least tangentially, it might be best to wait for that PR to go in, after which we can operate on the improved and clarified threading paradigm it will introduce. Thoughts?